src/Pokeview/components/Type.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Type.scss';
+
+const Type = ({name, type}) => {
```

Comment:
name is also not necessary, you can use the type prop, it doesn't matter if it is in english

Reply:
Fixed